plotly/tools.py

```diff
@@ -2274,6 +2310,160 @@ def create_candlestick(open, high, low, close,
+        (from multiple datasets) can be created in the same plot.
+
+        :param (list) hist_data: list of histogram data, should be a list of
+            lists if multiple data sets are plotted on the same plot
+        :param (list) group_labels: list of strings of the name of each data
+            set
+        :param (float) bin_size: size of histogram bin, default = 1.
+        :param (string) curve_type: 'kde' or 'normal', default = 'kde'
+        :param (bool) show_hist: True/False defines if histogram is added to
+            distplot, default = True
+        :param (bool) show_curve: True/False defines if curve is added to
+            distplot, default = True
+        :param (bool) show_rug: True/False defines if rug plot is added to
+            distplot, default = True
+        :param (list) colors: list of strings of color values,
+            default = first 10 Plot.ly colors
+        :param (list) rug_text: list of strings of hovertext for rug_plot,
+            default=None
+
+        :rtype (dict): returns a representation of a distplot figure.
+
+        Example 1: Simple distplot of 1 data set
+        ```
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+
+        hist_data = [[1.1, 1.1, 2.5, 3.0, 3.5,
+                      3.5, 4.1, 4.4, 4.5, 4.5,
+                      5.0, 5.0, 5.2, 5.5, 5.5,
+                      5.5, 5.5, 5.5, 6.1, 7.0]]
+
+        group_labels = ['distplot example']
+
+        fig = FF.create_distplot(hist_data, group_labels)
+
+        url = py.plot(fig, filename='Simple distplot', validate=False)
+        ```
+
+        Example 2: Two data sets and added rug text
+        ```
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+
+        # Add histogram data
+        hist1_x = [0.8, 1.2, 0.2, 0.6, 1.6,
+                   -0.9, -0.07, 1.95, 0.9, -0.2,
+                   -0.5, 0.3, 0.4, -0.37, 0.6]
+        hist2_x = [0.8, 1.5, 1.5, 0.6, 0.59,
+                   1.0, 0.8, 1.7, 0.5, 0.8,
+                   -0.3, 1.2, 0.56, 0.3, 2.2]
+
+        # Group data together
+        hist_data = [hist1_x] + [hist2_x]
+
+        group_labels = ['2012', '2013']
+
+        rug_text = ['a', 'b', 'c', 'd', 'e',
+                    'f', 'g', 'h', 'i', 'j',
+                    'k', 'l', 'm', 'n', 'o']
+
+        # Create distplot
+        fig = FF.create_distplot(
+            hist_data, group_labels, rug_text=rug_text, bin_size=.2)
+
+        # Add title
+        fig['layout'].update(title='Dist Plot')
+
+        # Plot!
+        url = py.plot(fig, filename='Distplot with rug text', validate=False)
+        ```
+
+        Example 3: Plot with normal curve and hide rug plot
+        ```
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+
+        x1 = np.random.randn(20)
+        x2 = np.random.randn(40)+1
+        x3 = np.random.randn(10)-1
+        x4 = np.random.randn(10)+2
```

Comment:
could we move these up to size `200` and bin size `0.2`? It's so much more pretty :)

![image](https://cloud.githubusercontent.com/assets/1280389/9557209/2f032878-4da8-11e5-8a2f-16bc8142d400.png)